.circleci/workflows.yml

```diff
@@ -87,10 +87,13 @@ workflows:
             - Initial Setup
             - Build Help and Test Files
             - Build Nuspec
+            - Invoke Git Commit
           context: aws-credentials
       - Invoke Git Commit:
           requires:
-            - Module Validation Tests
+            - Initial Setup
```

Comment:
We flipped the git commit to occur before Module Validation Tests, this will write back the changelog + module banner + docs to the repo before running tests (which would always fail if a new version of PSModule was specified). Switching the order should now always write back a new changelog, module banner and docs, the Module Validation will fail on the first round but at least the dev will have the files they need to change to pass module validation tests next commit.